support/views.py

```diff
@@ -1,29 +1,41 @@
 from rest_framework import generics
+from django_filters import rest_framework
+
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.authentication import (
-    SessionAuthentication, BasicAuthentication
+    SessionAuthentication,
+    BasicAuthentication
 )
 
-from .models import (LayersGroup, CategoryLayersGroup)
-from .serializers import (LayersGroupSerializer, CategoryLayersGroupSerializer)
+from support import (
+    models,
+    serializers,
+    filters as support_filters
+)
 
 
 class AuthModelMixIn:
-    """ AuthModelMixIn default class for `support.views` """
+    """AuthModelMixIn default class for `support.views`."""
 
     authentication_classes = (SessionAuthentication, BasicAuthentication)
```

Comment:
And here, use `authentication_classes = (authentication.SessionAuthentication, authentication.BasicAuthentication)`

Reply:
ok